### ISSUE TITLE
Mismatched publication date in blog file name

### ISSUE THREAD

**Issue by dipesh-rawat** [kubernetes/website#46399]
## **This is a Bug Report**

A recent blog file name ([`2024-04-28-contribute-as-sig-docs-reviewer.md`](https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md)) does not accurately reflect the publication date.

### **Problem:**

A recent _mirrored_ blog file name [2024-04-28-contribute-as-sig-docs-reviewer.md](https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md) does not reflect the actual publication date and may cause confusion. To maintain accuracy and clarity, we need to update the file name to reflect correct pulbished date (2024-05-16). This will ensure that our file naming conventions are up-to-date and align with our publication practices.

### **Proposed Solution:**

We should update the [filename](https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md) to reflect the correct publication date.

To resolve this issue, follow these steps to update the blog file name with the correct publication date:

- Locate the existing blog file named **`2024-04-28-contribute-as-sig-docs-reviewer.md`** [(link here)](https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md).

 - Rename the file to `2024-05-16-contribute-as-sig-docs-reviewer.md` to match the current publication date.
 ```diff
 - Current Name: content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md
 
+  Updated Fixed Name: content/en/blog/_posts/2024-05-16-contribute-as-sig-docs-reviewer.md
```

###  **Page to Update:**

- **Markdown source:** [2024-04-28-contribute-as-sig-docs-reviewer.md](https://github.com/kubernetes/website/blob/52ebe0472501fcd51ab84e071ac46f2ebca4a585/content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md)

- **Blog link:** https://kubernetes.io/blog/2024/05/16/contribute-as-sig-docs-reviewer/

### **Additional Information:**
/kind bug
/priority backlog
/language en
/area blog
/triage accepted
/good-first-issue

**Comment by dipesh-rawat** [comment]
If you have previous experience with contributing to Kubernetes documentation, kindly leave this issue available for new contributors to tackle.

> ℹ️ This task is ideal for a new contributor to pick up as it's a straightforward fix that introduces them to the repository's structure and contribution process. If you have already contributed to Kubernetes documentation in the past, we encourage you to give others the opportunity to get involved. This task presents an excellent entry point for newcomers to make their initial contribution to the project!

**Comment by ricardoamaro** [comment]
Thank you for creating this @dipesh-rawat !

**Comment by Shubham82** [comment]
closing this issue, as it is resolved by PR #46411
Thanks!
/close
